.changeset/quiet-grapes-end.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
```

Comment:
question: We're OK making this a minor change even though we're removing prop?

Reply:
According to [Grokt](https://grokt.shopify.io/results?q=disable1Password), it's only used within Polaris so is safe to remove